tests/test_fuzzy.py

```diff
@@ -200,25 +202,31 @@ def test_definition(self):
         fuzz = fuzzy.FuzzyFloat(2.0, 3.0)
         for _i in range(20):
             res = utils.evaluate_declaration(fuzz)
-            self.assertTrue(2.0 <= res <= 3.0, "value %d is not between 2.0 and 3.0" % res)
+            self.assertTrue(
+                2.0 <= res <= 3.0, "value %d is not between 2.0 and 3.0" % res
```

Comment:
If split across lines, it would be easier to read with the error message on a dedicated line.